cmd/agent/subcommands/snmp/command.go

```diff
@@ -190,12 +190,11 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 					LogParams:    log.ForOneShot(command.LoggerName, "off", true)}),
 				core.Bundle(),
 				aggregator.Bundle(),
-				forwarder.Bundle(),
+				forwarder.BundleWithProvider(defaultforwarder.NewParams),
```

Comment:
Should we use `forwarder.Bundle(defaultforwarder.NewParamsWithResolvers),`?

Reply:
I kept the existing behaviour which used [defaultforwarder.NewParams](https://github.com/DataDog/datadog-agent/pull/28465/files#diff-5c62ee07a306060f69fad02447ebbd82f9fef0c75a846a6a2fd865d8bdd3eb64L198). Any reason to use `defaultforwarder.NewParamsWithResolvers` instead?

Reply:
Sorry I meant `forwarder.Bundle(defaultforwarder.NewParams(....))`

Reply:
Please note that `NewParams` type is `func(config config.Component, log log.Component) Params` that is why I used `BundleWithProvider` instead of `Bundle`.